ncm-ofed: no loading of CAF::Service

Running ofed component returns :
 [ERROR] component ofed executing method Configure fails: Can't locate object method "new" via package "CAF::Service" (perhaps you forgot to load "CAF::Service"?) at /usr/lib/perl/NCM/Component/ofed.pm line 102.

--- a/ncm-ofed/src/main/perl/ofed.pm
+++ b/ncm-ofed/src/main/perl/ofed.pm
@@ -9,6 +9,9 @@ NCM::Component::ofed - OFED configuration component
 use parent qw(NCM::Component);
 our $EC = LC::Exception::Context->new->will_store_all;
 
+our $NoActionSupported = 1;
+
+use CAF::Service;
 use EDG::WP4::CCM::TextRender;
 use Readonly;
 
@@ -29,7 +32,7 @@ sub _render
                                                  log => $self);
 
     my $cfg_fh = $cfg_trd->filewriter($cfg_fn, log => $self, backup => ".old", mode => oct(644));
-    if(! defined($cfg_fh)) {
+    if (!defined($cfg_fh)) {
         $self->error("Failed to render $cfg_fn: $cfg_trd->{fail}");
         return;
     } else {

